server/cluster.go

```diff
@@ -119,11 +121,10 @@ func (c *RaftCluster) start() error {
 	var classifier namespace.Classifier
 	if c.s.cfg.EnableNamespace {
```

Comment:
I prefer using classifier name switch later, not EnableNamespace.

Reply:
Agree. But I prefer to keep this PR's intent accurate. We can do it later.